src/gracenote.ts

```diff
@@ -24,7 +24,7 @@ export class GraceNote extends StaveNote {
     return 0.66;
   }
 
-  constructor(note_struct: GraceNoteStruct) {
+  constructor(note_struct: Partial<GraceNoteStruct>) {
```

Comment:
Sometimes I see note_struct and sometimes it is noteStruct. Should we standardize to `noteStruct` where possible? Maybe the underscores should be reserved for keys of parameter objects, or legacy public fields.

Reply:
Agree, it will add a few files though